grc.fraudDetection_web/src/main/java/com/typedb/examples/fraud/db/AddressDao.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2022 Vaticle
```

Comment:
I also noticed that, for example, an `AddressDao` represents a _data store_ containing (potentially) multiple Addresses.

Given this, does it make sense to name it `AddressesDao`? Elsewhere in the codebase we have a variable named `banks` of type `BankDao`. Or is `Dao` a commonly understood term that represents a collection?

Reply:
Dao is the commonly accepted term for "data access object", and I believe it is always singular unless it always returns a collection. For example, more complete dao would be able to insert/update/get a single entity. We simply didn't have the need to create them, and only added what we needed to support a few GraphQL queries.

Reply:
In short, you can think of things like this as areas we left incomplete since this is just an example